tests/test_views_buckets.py

```diff
@@ -227,7 +227,10 @@ def test_buckets_can_be_deleted(self):
     def test_every_collections_are_deleted_too(self):
         self.app.put_json(self.bucket_url, MINIMALIST_BUCKET,
                           headers=self.headers)
-        self.app.get(self.collection_url, headers=self.headers, status=404)
+        response = self.app.get(self.collection_url, headers=self.headers, status=404)
+        self.assertEqual(response.json['details']['id'], 'barley')
+        self.assertEqual(response.json['details']['resource_name'], 'collection')
```

Comment:
We already covered this in `test_unknown_collection_raises_404` from `test_views_collections`. The assertion is not necessary here.